public/app/features/correlations/Forms/ConfigureCorrelationSourceForm.tsx

```diff
@@ -20,6 +21,10 @@ const getStyles = (theme: GrafanaTheme2) => ({
     font-family: ${theme.typography.fontFamilyMonospace};
     font-weight: ${theme.typography.fontWeightMedium};
   `,
+  // we alternate background colors, make card slightly darker than either
+  cardContainerBackground: css`
```

Comment:
This is non standard, but if we use a set color instead of a color with an opacity, the background color disappears on dark colored rows in the interactive table. With opacity, the background color is always visible as a slightly darker version of the background.